CHANGES.rst

```diff
@@ -1,6 +1,11 @@
 Changes
 =======
 
+Version 0.7.0 (Unreleased)
```

Comment:
I'm not sure we'll be releasing 0.7, it could be 1.0 in case of dramatic rewrites.  Also, we could have several development branches, e.g. `master` and `next`, so we cannot be always sure about which version will be the next.

A safer default to use might be: 

```
Version master (UNRELEASED)
```

Reply:
One more thing: if you are adding `CHANGES.rst` already, then please compile main news for the past, so that we don't forget.  Does not have to be ultra clean, just some quick lines compiled from:

```console
$ git log v0.6.0.. --format=oneline
```

Having main points already now would help with showing proper changes in RTFD already.  (The recent PR would show only Black, which would be misleading.)

We can always clean the verbiage better before releasing REANA v0.7.0.